application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsDashboardTemplateProvider.xml

```diff
@@ -0,0 +1,255 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<xwikidoc version="1.4" reference="Analytics.Code.AnalyticsDashboardTemplateProvider" locale="">
```

Comment:
You have 2 template providers. I suppose the second one is the one you wanted to commit, I added comments on that one